crates/solvers-dto/src/auction.rs

```diff
@@ -33,11 +34,69 @@ pub struct Order {
     #[serde_as(as = "HexOrDecimalU256")]
     pub sell_amount: U256,
     #[serde_as(as = "HexOrDecimalU256")]
+    pub full_sell_amount: U256,
+    #[serde_as(as = "HexOrDecimalU256")]
     pub buy_amount: U256,
+    #[serde_as(as = "HexOrDecimalU256")]
+    pub full_buy_amount: U256,
+    pub fee_policies: Option<Vec<FeePolicy>>,
+    pub valid_to: u32,
     pub kind: Kind,
+    pub receiver: Option<H160>,
+    pub owner: H160,
     pub partially_fillable: bool,
+    pub pre_interactions: Vec<InteractionData>,
+    pub post_interactions: Vec<InteractionData>,
+    pub sell_token_balance: SellTokenSource,
```

Comment:
There are still a few cases of `sell_token_balance` instead of `sell_token_source`.